tests/conftest.py

```diff
@@ -63,25 +35,30 @@ def _verify_url(request, server_url, user, org):
 
 
 @pytest.fixture(name="server_url", scope="session")
-def fixture_server_url(request):
+def fixture_server_url(request) -> typing.Any:
     """Return the URL to the Redash server."""
-    return request.config.option.server_url
+    return request.config.getoption("server_url")
 
 
 @pytest.fixture(name="org", scope="session")
-def fixture_org():
+def fixture_org() -> str:
     """Return the slug of an org."""
     return "default"
 
 
 @pytest.fixture(name="unknown_user")
-def fixture_unknown_user(variables, org):
+def fixture_unknown_user(variables: typing.Dict, org: str) -> User:
     """Return a user that is not registered."""
     return User(**variables[org]["users"]["unknown"])
 
 
 @pytest.fixture(name="user", scope="session")
-def fixture_user(create_user, variables, org, user_factory):
+def fixture_user(
+    create_user: typing.Callable[..., User],
+    variables: typing.Dict,
+    org: str,
+    user_factory: UserFactory,
+) -> User:
     """Return a registered user."""
 
```

Comment:
Does this space need to be there?

Reply:
I used **black** to auto-format the code.